beacon-chain/sync/subscriber_data_column_sidecar.go

```diff
@@ -52,3 +54,30 @@ func (s *Service) receiveDataColumnSidecar(ctx context.Context, sidecar blocks.V
 
 	return nil
 }
+
+// allDataColumnSubnets returns the data column subnets for which we need to find peers
+// but don't need to subscribe to. This is used to ensure we have peers available in all subnets
+// when we are serving validators. When a validator proposes a block, they need to publish data
+// columns on all subnets. This method returns a nil map when there is no validators custody
```

Comment:
```suggestion
// column sidecars on all subnets. This method returns a nil map when there is no validators custody
```

(Sorry to be picky here, but the mix between blob/blob sidecar and data column/data column sidecar created a lot of confusion in the past.)